ui/app/src/components/CrafterCMSNextBridge.tsx

```diff
@@ -23,11 +23,23 @@ import ThemeProvider from '@material-ui/styles/ThemeProvider';
 import { theme } from "../styles/theme";
 
 const cache = createIntlCache();
+const locale = document.documentElement.lang;
+export const intl = _createIntl(locale);
 
-export const intl = createIntl({
-  locale: 'en',
-  messages: {}
-}, cache);
+export async function _createIntl(locale: any) {
+  if (!locale) {
+    locale = 'en';
+    // TODO: throw console error
+  }
+  
+  return createIntl(
+    {       
+      locale,
+      messages: await import(`../translations/locales/${locale}.json`),
```

Comment:
Let’s discuss who to not delay rendering

Reply:
yep